apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js

```diff
@@ -166,24 +169,24 @@ const IdeaForm = ({
 									</Field>
 
 									<atoms.ToolTip
-										content="As an “idea owner” you’ll own the idea and be in charge of refine and update the information on the workshipping page."
+										content="As an “idea owner” you'll own the idea and be in charge of refine and update the information on the workshipping page."
 										direction="top"
 										delay={100}
 									>
 										<img alt='submit_image' src={popoverSvg} />
 									</atoms.ToolTip>
 								</atoms.Box>
-								<atoms.Typography type='p'style={{ marginLeft: '1rem', marginTop: '0.5rem', color:'#F03D3E', fontSize:'0.875rem' }}> {errors.involveLevel}</atoms.Typography>
+								<atoms.Typography type='p'style={{ marginLeft: '1rem', marginTop: '0.5rem', color:'#F03D3E', fontSize:'0.875rem' }}> {touched['involveLevel'] && errors.involveLevel}</atoms.Typography>
 							</atoms.Box>
 
 							<atoms.Typography type='p'>
 								After submitting your idea will be reviewed and enter the workshopping stage!
 							</atoms.Typography>
 
 							<atoms.Box style={{ fontSize: '1rem', alignItems:'center'}}>
-								<atoms.Checkbox disabled={false} required />
+								<input type="checkbox" style={{ height: '1.5rem', width:'1.5rem'}} required/>
```

Comment:
is there a style class we could use here instead of specifying `1.5rem` in a style attribute?

Reply:
We don't have that. And formik didn't accept styled-component well.